.circleci/config.yml

```diff
@@ -35,6 +35,20 @@ jobs:
           at: venv
       - run: make check_format
 
+  check_sdist:
+    docker:
+      - image: circleci/python:3.6
+    steps:
+      - checkout
+      - attach_workspace:
+          at: venv
+      - run: |
+          pip install -U setuptools
+          rm -rf dist  # make sure it's clean
+          python setup.py sdist > log.txt
+          python -c "from pathlib import Path;lines = Path('log.txt').read_text().splitlines(); assert any('copying LICENSE' in x for x in lines), lines"
+          python -c "from pathlib import Path;lines = Path('log.txt').read_text().splitlines(); assert any('copying requirements/main.txt' in x for x in lines), lines"
+
```

Comment:
@gwenzek might be worth writting this to the MAKEFILE somewhere? Not very familiar with how to make it work though
(and yes, it's ugly :s)